docs/next_steps.md

```diff
@@ -20,6 +20,56 @@ platform from the list of provisioned ones. Otherwise, when one machine is
 decommissioned, all others would have to be provisioned again after clearing
 Fobnail's memory.
 
+## Removing MAC address from platform metadata
+
+Currently, MAC address of the primary NIC is used as part of platform metadata
+described [here](https://fobnail.3mdeb.com/fobnail-api/#adminprovisionidmeta).
+Determining which NIC is the primary one has been problematic due to lack of
+standardized way to do so - procedure depends on how the NIC is connected, this
+should be PCI on virtually any x86 platform, but not on other architectures.
+
+- TODO: we could discuss other changes to platform metadata if so desired.
+
+## Recovery mechanism
+
+We need some mechanism to recover secrets in case attestation fails. Attestation
```

Comment:
In my opinion attestation-protected objects should be backed up _when they are created_. Right now we don't create anything on the Token (other than key that isn't later used for anything), so every secret comes from PC, from which it can be copied to secure backup storage before it is purged from the platform. Having a way of getting the secrets from unprovisioned (as far as Token is concerned) platform is another potential point of attack.